frontend/src/stories/TeamsTable.stories.js

```diff
@@ -12,4 +12,5 @@ const Template = args => <TeamsTable {...args} />
 export const Primary = Template.bind({});
 Primary.args = {
   teams: user.teamMemberships,
+  authUser: user
```

Comment:
What's the difference between  "./fixtures/user.json" and  "./fixtures/authUser.json"?

Please make sure you pass the right object in.

The authUser is the user that is currently authenticated.
The user if the person we are looking at. 

So if Kaleem logs in and looks at Babalwa's info, then the authuser is Kaleem and the the user is Babalwa.

The data structures are different.